### DISCUSSION TITLE
Disable Youtube Shorts in latest update

### DISCUSSION THREAD

**Discussion by crayonware** [PoomSmart/YTABConfig#118] [General]
I've noticed that shorts have popped up in my home page again after installing the newest update. After spending an embarrassing amount of time watching brainrot, I decided to make another list of A/B flags to disable shorts. Hopefully this can be added as a feature in a future update :)

`YTColdConfig.enableResumeToShorts: 0
YTColdConfig.enableShortsTargeted: 0
YTColdConfig.iosEnableWatchToShortsTransition: 0
YTColdConfig.mainAppCoreClientIosRemoveShortsInPivotFlagForIphoneKillswitch: 1
YTColdConfig.shortsConsumptionClientGlobalConfigIosEnableShortsProminentNavigationBar: 0`

**Comment by gingerbeardman** [comment]
I've tried these settings but the app still opens to Shorts on my Home.

**Comment by gingerbeardman** [comment]
Seems it did work... I needed to hide the Shorts that were already on screen, using the triple dot button in the top right corner of each one. Since then, they've not returned!

**Comment by foxnoodles** [comment]
Does not work for me. They keep coming back

**Comment by crayonware** [comment]
I recommend using YTLite+ as there is a dedicated option to remove shorts,
an option to show shorts as regular videos and a limit to how many shorts
can be viewed at a time.

**Reply by foxnoodles** [reply]
Yes I’m aware of that but I’m looking for a standalone tweak solution and not a modded ipa. 

I just used a tweak called  UnShorts to remove shorts but keep the shorts tab. (Also modified to remove some conflicts to keep the shorts progress bar tweak installed)

**Comment by crayonware** [comment]
Is there any reason why you don't want a modified IPA? Are you just using
the standard Youtube app? Anyways, you could try YTNoShorts. Also take a
look at
https://github.com/dayanch96/YTLite/blob/fc8e5daf7bdc2b11c36ad3ac02cc83b6064e4f94/YTLite.x#L46
and
https://github.com/dayanch96/YTLite/blob/fc8e5daf7bdc2b11c36ad3ac02cc83b6064e4f94/YTLite.x#L701

https://github.com/arichornlover/YTNoShorts

Good luck, and let me know if it works out!

**Comment by iMonZ** [comment]
Thanks it worked! Although I could not find the last entry.

**Comment by iMonZ** [comment]
How can I hide the shorts button at home?
Since the Tweaks from uYou are not working.

**Reply by gingerbeardman** [reply]
1. Hide the Shorts in YTAB Settings
2. The section will still be there
3. Hide each short individually then the section will disappear
4. Repeat if it comes back

**Reply by iMonZ** [reply]
> 1. Hide the Shorts in YTAB Settings
> 2. The section will still be there
> 3. Hide each short individually then the section will disappear
> 4. Repeat if it comes back

Hey thanks, and that worked!
But I mean how to hide the button at the bottom Navigation Bar

**Reply by gingerbeardman** [reply]
There's an option to replace it, though sadly I can't remember which option and where.

But my current setup shows:

- Home
- Explore
- Upload
- Subscriptions
- You
- uYou

**Reply by iMonZ** [reply]
> There's an option to replace it, though sadly I can't remember which option and where.
> 
> But my current setup shows:
> 
> * Home
> * Explore
> * Upload
> * Subscriptions
> * You
> * uYou

Sadly my Shorts Button is still visible :(

**Reply by iMonZ** [reply]
> > There's an option to replace it, though sadly I can't remember which option and where.
> > But my current setup shows:
> > 
> > * Home
> > * Explore
> > * Upload
> > * Subscriptions
> > * You
> > * uYou
> 
> Sadly my Shorts Button is still visible :(

Nevermind. https://github.com/YTLitePlus/YTLitePlus fixed this
